src/sections/Learn-Layer5/Learn-Components/Card-Wrapper/index.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import CardComponentWrapper from "./Card-Wrapper.component.style";
+import CardComponentWrapper from "./card-wrapper.style";
```

Comment:
What I meant was instead of creating a new file add this code in the `src/sections/Learn-Layer5/index.js` file itself., not to move the file.

Reply:
Ohk ..will be rectifying in a moment.. 👍🏻